app/views/dashboard/index.html.erb

```diff
@@ -14,64 +14,96 @@
         </div>
       <% end %>
       <div class="card-body">
-        <table data-toggle="table" class="table">
+        <%= link_to "Active machines (#{@vms&.size}) <small>&#x25bc;</small>".html_safe, '#active' %><br />
+        <%= link_to "To be archived (#{@pending_archivation_vms&.size}) <small>&#x25bc;</small>".html_safe, '#to_be_archived', class: "#{"text-danger" unless @pending_archivation_vms&.empty?}"%><br />
+
+        <h2 id="active">Active machines</h2>
+        <table data-toggle="table" class="table" id="vms_active">
           <thead>
           <tr>
-
+            <th scope="col" data-sortable="true" class="bg-primary text-white rounded-left">Name</th>
+            <th scope="col" data-sortable="true" class="bg-primary text-white">Project</th>
+            <th scope="col" data-sortable="true" class="bg-primary text-white">Responsible</th>
+            <th scope="col" data-sortable="true" class="bg-primary text-white rounded-right">Status</th>
           </tr>
+          </thead>
           <tbody id="vms">
-          <% @vms.each do |vm| %>
-            <tr>
-              <td><%= vm.name %></td>
-              <td><%= vm.boot_time %></td>
-              <td><%= button_to "Details",
-                                {controller: :vms, action: 'show', id: vm.name},
-                                method: :get, class: "btn btn-success" %></td>
-              <td><%= button_to "Delete",
-                                {controller: :vms, action: 'destroy', id: vm.name},
-                                method: :delete, class: "btn btn-danger" %></td>
+          <% @vms&.each do |vm| %>
```

Comment:
Why is there an `&`?

Reply:
`each` will only be called if vms is not null.

Reply:
Ok, thanks :)